pkg/scheduler/framework/plugins/noderesources/balanced_allocation.go

```diff
@@ -73,6 +85,15 @@ func (ba *BalancedAllocation) PreScore(ctx context.Context, cycleState fwk.Cycle
 	state := &balancedAllocationPreScoreState{
 		podRequests: podRequests,
 	}
+	if ba.enableDRAExtendedResource && state.draPreScoreState == nil {
```

Comment:
`state.draPreScoreState` is always nil here
```suggestion
	if ba.enableDRAExtendedResource {
```